src/enums/experience.ts

```diff
@@ -138,6 +145,8 @@ export interface ExperienceInput {
   viewState: InitialViewState;
   /** experience purposes to be added */
   experiencePurposeInputs: ExperiencePurposeInput[];
+  /** set of IAB signals to be communicated in this experience */
+  iabSignals: IABSignal[];
```

Comment:
I suppose we need this to be an array because a customer might need both USP and US CA, for instance?